src/com/projectkorra/projectkorra/earthbending/lava/LavaSurge.java

```diff
@@ -236,15 +237,15 @@ public void progress() {
 				x = (this.random.nextBoolean()) ? -x : x;
 				z = (this.random.nextBoolean()) ? -z : z;
 
-				fbs.setVelocity(this.direction.clone().add(new Vector(x, 0.2, z)).multiply(1.2));
+				GeneralMethods.setVelocity(this,fbs, this.direction.clone().add(new Vector(x, 0.2, z)).multiply(1.2));

```

Comment:
Please change to
`GeneralMethods.setVelocity(this, fbs, this.direction.clone().add(new Vector(x, 0.2, z)).multiply(1.2));`